src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -799,6 +803,22 @@ impl Edges {
         detached.extend(detached_source_iter);
         detached.into_iter()
     }
+
+    pub fn node_attached_edges(&self, node_id:NodeId) -> Vec<EdgeId> {
```

Comment:
the name of the function doesnt tell me what it does. It definitely needs better name like `attached_edges_to_node`